decryptor/mysql/prepared_statements.go

```diff
@@ -187,7 +187,11 @@ func (m *mysqlBoundValue) Copy() base.BoundValue {
 
 // SetData set new value to BoundValue using ColumnEncryptionSetting if provided
 func (m *mysqlBoundValue) SetData(newData []byte, setting config.ColumnEncryptionSetting) error {
-	m.textData = newData
+	// means that we set encrypted data
```

Comment:
hm... for now it is used by QueryEncryptor uses it after encryption/tokenization. but what about if some else component will use it and there will be not only encrypted? can we somehow check that it is encrypted and we should change type?